pallets/token-manager/src/lib.rs

```diff
@@ -574,6 +586,22 @@ pub mod pallet {
 
             return Ok(())
         }
+
+        #[pallet::call_index(10)]
+        #[pallet::weight(<T as pallet::Config>::WeightInfo::toggle_lowering())]
+        pub fn toggle_lowering(origin: OriginFor<T>, enabled: bool) -> DispatchResult {
```

Comment:
I prefer having a `set_config` function where different values can be configured
we already have `set_lower_schedule_period` so it would be better if we have one set admin setting function.

Reply:
I personally prefer having smaller extrinsics that do specific tasks. They are easier to test, benchmark and reason with. Happy to hear other thoughts

Reply:
For me, with the little experience I have, I feel like smaller extrinsics are easier to understand but happy either way!

Reply:
I'm afraid I have to disagree, multiple entry points for sudo configuration is a security concern.
The codebase increases significantly if you have to replicate all origin and benchmark tests for each one of the entrypoints, making it difficult to navigate through files.
This is a common practice in many pallet.
Alternatively we can make this a const, defined by the runtime. Its clean and minimal and the way many pallets are configured in substrate.